core-primitives/enclave-api/ffi/build.rs

```diff
@@ -16,27 +16,29 @@
 
 */
 
-use std::env;
-
 fn main() {
-	let sdk_dir = env::var("SGX_SDK").unwrap_or_else(|_| "/opt/intel/sgxsdk".to_string());
-	let is_sim = env::var("SGX_MODE").unwrap_or_else(|_| "HW".to_string());
-
-	// NOTE: if the crate is a workspace member rustc-paths are relative from the root directory
-	println!("cargo:rustc-link-search=native=./lib");
-	println!("cargo:rustc-link-lib=static=Enclave_u");
-
-	println!("cargo:rustc-link-search=native={}/lib64", sdk_dir);
-	println!("cargo:rustc-link-lib=static=sgx_uprotected_fs");
-	match is_sim.as_ref() {
-		"SW" => {
-			println!("cargo:rustc-link-lib=dylib=sgx_urts_sim");
-			println!("cargo:rustc-link-lib=dylib=sgx_uae_service_sim");
-		},
-		_ => {
-			// HW by default
-			println!("cargo:rustc-link-lib=dylib=sgx_urts");
-			println!("cargo:rustc-link-lib=dylib=sgx_uae_service");
-		},
+	if cfg!(feature = "link-sgx-libs") {
```

Comment:
Unfortunately, we can't use `cfg(test)` because it doesn't rerun the build.rs. We need to use an actual feature flag.